### PR TITLE
Update Radio Programming for 2022

### PR DESCRIPTION
**Pull request by Kevin-OConnor**
Updates for 2022 and addresses some feedback from Beta.

### CODE REVIEW

**Review by Daltz333 — DISMISSED** [review]

**Comment by sciencewhiz** [comment]
Similar changes should be made to programming-radios-for-fms-offseason

File: source/docs/software/driverstation/programming-radios-for-fms-offseason.rst [hunk 1]
```diff
@@ -5,16 +5,16 @@ Programming Radios for FMS Offseason
...
-
 Pre-Requisites
 --------------
 
 Install the FRC\ |reg| Radio Configuration Utility software per the instructions in :ref:`Programming your radio <docs/zero-to-robot/step-3/radio-programming:Prerequisites>`
 
+Before you begin using the software:
+
+#. Disable WiFi connections on your computer, as it may prevent the configuration utility from properly communicating with the bridge
+#. Plug directly from your computer into the wireless bridge ethernet port closest to the power jack and make sure no other devices are connected to your computer via ethernet. If powering the radio via PoE, plug an Ethernet cable from the PC into the socket side of the PoE adapter (where the roboRIO would plug in). If you experience issues configuring through the PoE adapter you may try connecting the PC to the alternate port on the radio.
```
Comment (Daltz333) [inline source/docs/software/driverstation/programming-radios-for-fms-offseason.rst:16]:
```suggestion
#. Plug directly from your computer into the wireless bridge ethernet port closest to the power jack. Make sure no other devices are connected to your computer via ethernet. If powering the radio via PoE, plug an Ethernet cable from the PC into the socket side of the PoE adapter (where the roboRIO would plug in). If you experience issues configuring through the PoE adapter, you may try connecting the PC to the alternate port on the radio.
```

Comment (Daltz333) [reply source/docs/software/driverstation/programming-radios-for-fms-offseason.rst:16]: I think having them as two separate sentences reads a bit better.

File: source/docs/zero-to-robot/step-3/radio-programming.rst [hunk 2]
```diff
@@ -5,30 +5,28 @@ Programming your Radio
...
+`FRC Radio Configuration 22.0.1 Israel Version <https://firstfrc.blob.core.windows.net/frc2022/Radio/FRC_Radio_Configuration_22_0_1_IL.zip>`_
 
 .. note:: The _IL version is for Israel teams and contains a version of the OM5PAC firmware with restricted channels for use in Israel.
 
 .. note::. Teams planning on using the radio tool on a machine without the WPILib suite installed, will need a copy of Java installed. This can be downloaded `here <https://adoptopenjdk.net/archive.html?variant=openjdk11&jvmVariant=hotspot>`__.
 
+Before you begin using the software:
+
+1. :ref:`Disable all other network adapters <docs/networking/networking-introduction/roborio-network-troubleshooting:Disabling Network Adapters>`
+2. Plug directly from your computer into the wireless bridge ethernet port closest to the power jack and make sure no other devices are connected to your computer via ethernet. If powering the radio via PoE, plug an Ethernet cable from the PC into the socket side of the PoE adapter (where the roboRIO would plug in). If you experience issues configuring through the PoE adapter you may try connecting the PC to the alternate port on the radio.
```
Comment (Daltz333) [inline source/docs/zero-to-robot/step-3/radio-programming.rst:26]:
```suggestion
2. Plug directly from your computer into the wireless bridge ethernet port closest to the power jack. Make sure no other devices are connected to your computer via ethernet. If powering the radio via PoE, plug an Ethernet cable from the PC into the socket side of the PoE adapter (where the roboRIO would plug in). If you experience issues configuring through the PoE adapter, you may try connecting the PC to the alternate port on the radio.
```

**Review by Daltz333 — CHANGES_REQUESTED** [review]

**Pull request merged**